app/src/main/java/org/mozilla/fenix/compose/SelectableChip.kt

```diff
@@ -41,6 +43,8 @@ import org.mozilla.fenix.theme.Theme
 fun SelectableChip(
     text: String,
     isSelected: Boolean,
+    selectedTextColor: Color? = null,
+    unselectedTextColor: Color? = null,
     selectedBackgroundColor: Color? = null,
     unselectedBackgroundColor: Color? = null,
```

Comment:
More of a follow up, but when I see multiple color pairings and a growing param list, this reminds me of `ButtonDefaults.textButtonColors()` that we saw in `TextButton`. See https://cs.android.com/androidx/platform/tools/dokka-devsite-plugin/+/master:testData/compose/source/androidx/compose/material/Button.kt;l=235?q=textButtonColors()&ss=androidx

Perhaps as a followup, it would be appropriate to explore doing something similar.

Reply:
That said this doesn't have to happen immediately, but definitely worthwhile to have it filed by converting the above comment into an issue.

Reply:
I'll make a follow-up to condense the entire implementation into one parameter for each. Separately, I think the notion of a "selectable" chip seems redundant when we can just have a top-level `Chip` component and the implementor handle any state logic.

Reply:
I am not sure I fully follow about the latter, but I feel like I will need to see some sample code to fully comprehend. 

On a different note, I see we aren't using FenixTypography here, so another potential followup or many followup to audit all our Text() usages to ensure they are using a typography.

Reply:
Opened:
- https://github.com/mozilla-mobile/fenix/issues/27401